### DISCUSSION TITLE
Error after deleting user groups

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6741] [Q&A] [answered]
Bug description
Creating new group and assign some users to it then deleting that group make an issue

How to reproduce
1 - Login to the system as a super user

2 - Create a user group called “ New group “

3 - Add users to that group

4 - Delete the group

5 - Try to open one of the users that were assigned to that group

“ Call to a member function handle() on null ” error when trying to access one of the users accounts

Same error when trying to login with one of the users from the deleted group

**Comment by duncanmcclean** [comment, marked as answer]
For reference: a bug report has been created for this, #6740.

**Reply by HazemMohamed252** [reply]
> For reference: a bug report has been created for this, #6740.

Thanks
